reports/3_generate_report_emails.py

```diff
@@ -0,0 +1,98 @@
+# +
+import sys
+import os
+import pandas as pd
+from calitp.tables import tbl
+from siuba import *
+from postmarker.core import PostmarkClient
+import configparser
+
+# This script generates a email body using mjml. The result itself has templated variables
+# of form {{SOME_VARIABLE_NAME}}, so we will use jinja to complete these email bodies
+from jinja2 import Environment, StrictUndefined
+jinja_env = Environment(undefined=StrictUndefined)
+
+# +
+# read config from ini
+parser = configparser.ConfigParser()
+parser.read("config.ini")
+
+# TODO: how do people want to choose the config? Michael recommends using sys.argv
+#       so you could do e.g. python 3_generate_report_emails.py development
+config = parser["development"]
+
+# +
+## Generate the report and dev link base to create report url 
+
+# +
+# TODO: note this server token from environment. Michael recommends not putting it in the config,
+#       so we are not mixing sensitive info with configuration. Could also put it in a .env file that
+#       is .gitignored
+SERVER_TOKEN=os.environ["POSTMARK_SERVER_TOKEN"]
```

Comment:
Having 2 config files makes sense to me (similar to `.env` not being checked in, but `.env.dev` etc.. being checked in). 

Alternatively, if we use a `.env` file that is not committed, and python-dotenv, then you can put the sensitive info in the .env once, which should cut down the risk of the key getting copied back to the example (since the secrets are in a distinct config, the `.env`, which loads them as environment variables)

(Onboard with either / any approach)

Reply:
@Nkdiaz can you switch to use the `config.ini` and `config.ini-examples` approach above? Don't want to slow the PR with my comments :)